assets/js/googlesitekit/modules/components/DefaultModuleSettings.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Does this really need to be its own module? It's only being imported in one place and it's a no-op. I'd say it's fine to just pass `() => null` to https://github.com/google/site-kit-wp/pull/1671/files#diff-84bfda982eea3cfa46b67d1a68f5ffc7R182 and remove this file entirely.

Reply:
Part of the AC was to stub out this component and have it be finished in a follow up ticket.

>There should be a scaffolded JS component DefaultModuleSettings, within assets/js/googlesitekit/modules/components. For now it can just return null, implementation should be in a follow-up issue.